app/Http/Controllers/StudentsCouncil/EconomicController.php

```diff
@@ -85,41 +85,22 @@ public function payKKTNetreg(Request $request)
         $validator->validate();
 
         $payer = User::findOrFail($request->user_id);
+        // the current user will be the receiver
+        [$kkt, $netreg, $new_internet_expire_date] = $payer->payKKTNetreg(Auth::user()->id, $request->kkt, $request->netreg);
```

Comment:
What about writing a static method in the EconomicController class? That was where this code was originally written. I would rather write a method for it somewhere, so that we don't need to duplicate the code in the test. See the code I'm going to push soon.